nose/case.py

```diff
@@ -357,7 +359,7 @@ def __str__(self):
         name = "%s.%s.%s" % (self.cls.__module__,
                              self.cls.__name__,
                              name)
-        if arg:
+        if not arg == '()':
```

Comment:
No, `arg` as in returned at the top of the `__str__` function (`func, arg = self._descriptors()`), not `self.arg` which is the tuple of "real" args to the function.